biorxiv/journal_tracker/map_preprints_to_journals.py

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Following the Preprint to Published Path
+
+# The goal of this notebook is to map preprint dois to published dois and published dois to Pubmed Central articles.
```

Comment:
I think I'm behind on what your project is. I thought that you were looking into categories of preprints..so I'm not sure why you're comparing things here. Perhaps this will get added to the readme later though

Reply:
> Silly question, why are you mapping preprint doi to published doi as opposed to just published doi to articles?

The bioRxiv dump doesn't include published version dois. 🤷  For future analyses I need to know what preprints have been published and where; hence the necessary doi mapping step.

>I think I'm behind on what your project is. I thought that you were looking into categories of preprints..so I'm not sure why you're comparing things here. Perhaps this will get added to the readme later though

Whoops forgot to include that description. So this analysis was prompted after my committee meeting. Long story short I needed to take one step back and answer a fundamental question: "What is the difference between the bioRxiv corpus and Pubmed Central corpus?".  To answer this question I needed to compare word frequencies between corpora on a global scale (bioRxiv v PMC) and local scale (preprint version vs published versions).